help/develop/getting-started.md

```diff
@@ -196,6 +196,7 @@ package. Run `grok publish --release` so others could see your package, too.
 [custom filters]: https://github.com/datagrok-ai/public/blob/master/packages/Widgets/src/filters/radio_button_filter.js
 [Digital Signal Processing]: https://github.com/datagrok-ai/public/tree/master/packages/DSP
 [Biosignals]: https://github.com/datagrok-ai/public/tree/master/packages/BioSignals
+[Cheminformatics]: https://datagrok.ai/help/domains/chem/cheminformatics
```

Comment:
Let me ask a quick question here: we describe packages that can be found in our repo, but in this case, give a link to a wiki page.  Is this on purpose?

Reply:
I asked Dan what link we should provide here, and he suggested this link. Probably, we need to clarify this.

Reply:
Thanks a lot!